rules/windows/execution_command_shell_started_by_svchost.toml

```diff
@@ -1,39 +1,40 @@
 [metadata]
 creation_date = "2020/02/18"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2021/04/14"
 
 [rule]
 author = ["Elastic"]
 description = "Identifies a suspicious parent child process relationship with cmd.exe descending from svchost.exe"
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Svchost spawning Cmd"
 risk_score = 21
 rule_id = "fd7a6052-58fa-4397-93c3-4795249ccfa2"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.parent.name:svchost.exe and process.name:cmd.exe
+process where event.type in ("start", "process_started") and
+  process.parent.name : "svchost.exe" and process.name : "cmd.exe" and 
+  not process.command_line : "?:\\Windows\\SYSTEM32\\cmd.exe /c \"\"?:\\Program Files\\Npcap\\CheckStatus.bat\"\""
```

Comment:
where did this change come from?
seems a little to specific, could just leet users make exceptions

Reply:
its related to wireshark installs, very common and run a task every 5m (noisy).  (IIRC Paul also encountred/mentioned this while they were preparing for MITRE eval)